src/projector_monte_carlo_problem.jl

```diff
@@ -52,6 +52,8 @@ the [`FCIQMC`](@ref) algorithm.
 - `n_replicas = 1`: Number of synchronised independent simulations.
 - `replica_strategy = NoStats(n_replicas)`: Which results to report from replica
   simulations, see [`ReplicaStrategy`](@ref).
+- `spectral_strategy = GramSchmidt()`: The [`SpectralStrategy`](@ref) used for simulations
+  of spectral states.
```

Comment:
```suggestion
- `n_spectral = 1`: Number of targeted spectral states. Set `n_spectral  > 1` to find excited states.
- `spectral_strategy = GramSchmidt(n_spectral)`: The [`SpectralStrategy`](@ref) used for 
  orthogonalizing spectral states.
```
I would suggest to also implement it as such with introducing a new keyword argument `n_spectral` (similar to how it is done with replicas). This way the user doesn't have to wrap their head around a `spectral_strategy` if they just want a certain number of spectral states.

Reply:
Yes this is better, it's now implemented.